contracts/cw20-atomic-swap/src/contract.rs

```diff
@@ -239,7 +240,7 @@ fn query_details<S: Storage, A: Api, Q: Querier>(
 
     // Convert balance to human balance
     let balance_human = match swap.balance {
-        Balance::Native(coins) => BalanceHuman::Native(coins),
+        Balance::Native(cw0::Balance(coins)) => BalanceHuman::Native(coins),
```

Comment:
It would be nice to simplify this. Maybe:

`Balance::Native(coins) => BalanceHuman::Native(coins.0)`

OR

`Balance::Native(coins) => BalanceHuman::Native(coins.into())` (this may need another function)